manifests/config.pp

```diff
@@ -29,14 +29,27 @@
   $version = $::nexus::version,
 ) {
 
-  if $version !~ /\d.*/ or versioncmp($version, '3.0.0') >= 0 {
+  if $version !~ /\d.*/ or versioncmp($version, '3.1.0') >= 0 {
+    # Per the Sonatype documentation the custom nexus properties file is
+    # {karaf.data}/etc/nexus.properties where {karaf.data} is the work dir
+    $conf_path = 'etc/nexus.properties'
+    $nexus_properties_file = "${nexus_work_dir}/${conf_path}"
+  }
+  elsif versioncpm($version, '3.0.0') >= 0 {
```

Comment:
Typo versioncpm should be versioncmp, I believe this is why the tests are failing